in_toto/in_toto_record.py

```diff
@@ -14,166 +13,141 @@
   See LICENSE for licensing information.
 
 <Purpose>
-  Provides a command line interface to start and stop in-toto link metadata
-  recording.
+  Provides a command line interface to create a link metadata file in multiple
```

Comment:
What do you think of changing "multiple" to "two"?

Reply:
Hm. I was actually referring to the (potentially) multiple steps in between `start` and `record,`. But you are right, this is not clear. What about something like:
*"...create a link metadata file in two steps, in order to provide evidence for arbitrary activities. Note: If you want to provide evidence for a single command take a look at in-toto-record.*`?